Error when installing chaincode

When I follow the instructions to install chaincode (node install_chaincode.js) locally on Ubuntu I get an exception:

Error: Cannot find module 'stream-buffers'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/padraic/marbles/node_modules/fabric-client/lib/packager/Golang.js:23:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)

Any ideas to resolve?
Thanks
Padraic

That fixed that issue - thanks so much.

However when I try to issue the join channel command node test/integration/e2e/join-channel.js I initially get a timeout error. I updated the timeout value in app.js and re-run. 4 of 6 tests pass but I get an exception thrown
Error: Cannot find module '/home/padraic/marbles/scripts/fabric-sdk-node/test/fixtures/test/integration/e2e/join-channel.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
padraic@padraic-VirtualBox:~/marbles/scripts/fabric-sdk-node/test/fixtures$ sudo node test/integration/e2e/join-channel.js
sudo node test/integration/e2e/join-channel.js
sudo: node: command not found
padraic@padraic-VirtualBox:~/marbles/scripts/fabric-sdk-node/test/fixtures$ node test/integration/e2e/join-channel.js
node test/integration/e2e/join-channel.js
module.js:471
    throw err;

--- a/package.json
+++ b/package.json
@@ -15,8 +15,9 @@
     "express-session": "1.14.*",
     "fabric-ca-client": "1.0.0-alpha.0",
     "fabric-client": "1.0.0-alpha",
-	"pug": "2.0.0-beta11",
+    "pug": "2.0.0-beta11",
     "serve-static": "1.11.*",
+    "stream-buffers": "^3.0.1",
     "winston": "2.2.*",
     "ws": "1.1.*"
   },

